### PR TITLE
Launch cards

### PR DESCRIPTION
**Pull request by ElijahMSmith**
Added the cards to the /hackathon route that re-direct to other pages (event logistics, attendees, etc). Layout should be pleasant even on small screen sizes and the cards should wrap as needed.

### CODE REVIEW

File: src/components/hackathon/LaunchCard.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+interface LaunchCardProps {
+	icon: JSX.Element;
+	text: string;
+	onClick: () => void;
+}
+
+export default function LaunchCard({ icon, text, onClick }: LaunchCardProps) {
```
Comment (Victoria-Williamson) [inline src/components/hackathon/LaunchCard.tsx:7]: nothing to revise, just noting I like how u took in the props. Never thought you could do it that way! I will definitely be stealing that

File: src/pages/hackathon.tsx [hunk 2]
```diff
@@ -1,8 +1,45 @@
-import React from "react";
-import HackathonInfoBar from "../components/hackathon/HackathonInfoBar";
+import React from 'react';
+import HackathonInfoBar from '../components/hackathon/HackathonInfoBar';
+import LaunchCard from '../components/hackathon/LaunchCard';
+import { ReactComponent as LogisticsIcon } from '../assets/logistics.svg';
```
Comment (Victoria-Williamson) [inline src/pages/hackathon.tsx:4]: nit: could you put all these new SVGs into assets/Hackathon/ pls? I've been organizing the rest like that cause without that the folder will probably have 20+ assets atm. Plus I don't think any of these components will get reused.

**Review by Victoria-Williamson — CHANGES_REQUESTED** [review]
Looks pretty good; just a couple nit comments!

Also, I noticed a couple of minor UI issues with responsiveness and have some suggestions: 
- maybe make the card width a reusable size like md:w-1/3 for w-full. This way the cards should fill the width of the entire screen but also not be oddly small for phone-sized screens.
- add some padding onto the side of the page. 
- center the cards horizontally.

image

**Review by conroyr41 — APPROVED** [review]
Besides Victoria's suggestions, I think it looks good!

**Comment by ElijahMSmith** [comment]
Not 100% happy with the reactivity changes for the info bar at the top (not the launch cards) but it looks good IMO at all breakpoints, just not when changing between them per-se. Suggestions welcome, or if we're good with it for now then great!

**Review by Victoria-Williamson — APPROVED** [review]
It looks really good thanks! :)

**Pull request merged**